### ISSUE TITLE
Crashing when playing a song

### ISSUE THREAD

**Issue by seaweede** [Superjo149/auryo#122]
get this error occasionally on some tracks but not others. crashes the program after i click 'OK' or 'Copy Error'

`Unhandled Error
Error: Invalid pointer
Parameter 1 is null or undefined.
    at on (C:/Users/peecee/AppData/Local/Programs/auryo/resources/app.asar/webpack:/src/main/features/win32/win10/Win10MediaService.js:90:69)
    at EventEmitter.s.ipcMain.on.t (C:/Users/peecee/AppData/Local/Programs/auryo/resources/app.asar/webpack:/src/main/features/IFeature.js:58:30)
    at WebContents.<anonymous> (C:/Users/peecee/AppData/Local/Programs/auryo/resources/electron.asar/browser/api/web-contents.js:286:13)`

using win10 LTSB 64-bit, auryo portable 2.1.1

thanks for your hard work!

**Comment by Superjo149** [comment]
Possible fix is out, could you check if this is still occurring?

**Comment by isometriq** [comment]
I experienced a similar crash
```
Unhandled Error
TypeError: Cannot read property 'user' of undefined
    at on (C:/Users/isometriq/AppData/Local/Programs/auryo/resources/app.asar/webpack:/src/main/features/win32/win10/Win10MediaService.js:86:61)
    at EventEmitter.s.ipcMain.on.t (C:/Users/isometriq/AppData/Local/Programs/auryo/resources/app.asar/webpack:/src/main/features/IFeature.js:58:30)
    at WebContents.<anonymous> (C:/Users/isometriq/AppData/Local/Programs/auryo/resources/electron.asar/browser/api/web-contents.js:286:13)
```

To reproduce:

1. Start the app
2. Change the Windows volume with media keys (volume up/down on keyboard)
3. The Windows volume indicator appear with media buttons. It shows "Auryo No track is playing"
4. Press the play icon
5. Crash!

Reference: Windows volume indicator

![image](https://user-images.githubusercontent.com/4261000/46251917-387c2580-c42e-11e8-836e-7025257d88d7.png)

**Comment by Superjo149** [comment]
I'm pretty certain this will be fixed in 2.2, I'm rewriting everything to typescript which should smooth out some of these nasty undefined bugs.
